### DISCUSSION TITLE
Custom Summons Image

### DISCUSSION THREAD

**Discussion by tlkwok** [Lurkars/gloomhavensecretariat#394] [Q&A] [answered]
I would love to replace the summon thumbnail with my own custom image. However, I found that current summons images are referencing static images. Is it possible to replace the image url by importing data json?

**Comment by Lurkars** [comment, marked as answer]
Currently not possible, but I will add this. You may need to copy all summons of a char and adjust then, but at least should be possible! After added, I will document in #395  for you how to do it properly then.

**Reply by tlkwok** [reply]
Thanks!

**Comment by Lurkars** [comment]
@tlkwok happy if you can test/verify no #395
